### DISCUSSION TITLE
[info] OpenJS - upcoming conference news

### DISCUSSION THREAD

**Discussion by jorydotcom** [webdriverio/webdriverio#6353] [Community]
Wanted to share this FYI out to the webdriver.io community to let ya’ll know that the OpenJS Foundation has opened its call for speakers for the upcoming virtual OpenJS World event on 9 June 2021. We would love to make sure the project is well represented in the conference content! If anyone is interested and has more questions, wants help writing an abstract, or wants to get involved with the CFP Reviews, email me at jory @ thestoryofjory dot com or ping me in Slack.

We’d also love your help getting the word out about the event - if you’d be up to putting a link or banner on the project website, tweeting from your project account, etc. we would be very grateful!

**Comment by christian-bromann** [comment]
Hey Jory,
thanks for letting us know 😊 I am sure I can come up with a nice talk for the conference with a lot of updates about the project. Maybe something tailored about a specific area. Will see. Anyway happy to continue helping the committee with reviews and such.
